observability/otel/otel_rn/otel-rn-past-releases.adoc

```diff
@@ -8,6 +8,74 @@ toc::[]
 
 include::modules/otel-product-overview.adoc[leveloffset=+1]
 
+[id="otel-rn-3-2"]
+== Release notes for {OTELName} 3.2
+
+The {OTELName} is provided through the {OTELOperator}.
+
+////
+[id="otel-rn_3-2_cves"]
+=== CVEs
+
+This release fixes link:https://access.redhat.com/security/cve/cve-202?-?????[CVE-202?-?????].
+////
+
+[id="otel-rn_3-2_technology-preview-features"]
+=== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+* Host Metrics Receiver
+* OIDC Auth Extension
+* Kubernetes Cluster Receiver
+* Kubernetes Events Receiver
+* Kubernetes Objects Receiver
+* Load-Balancing Exporter
+* Kubelet Stats Receiver
+* Cumulative to Delta Processor
+* Forward Connector
+* Journald Receiver
+* Filelog Receiver
+* File Storage Extension
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="otel-rn_3-2_new-features-and-enhancements"]
+=== New features and enhancements
```

Comment:
Globally, there's new guidance to add _{context} to all non-0-level headings: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

Reply:
Since these were existing, based on your comment 0 in the PR, then I guess you don't need to worry about this. All new IDs need the _context

Reply:
@skrthomas, thank you, I've updated the files in this PR globally.